Core/clim-core/dialog-views.lisp

```diff
@@ -251,6 +251,36 @@
                                   :value-changed-callback #'value-changed-callback))))
 
 
+;;;; text-editor-view
```

Comment:
"implemetation" typo in the commit message

Reply:
Ok, I fixed it.